taskfiles/k8s.yml

```diff
@@ -17,6 +17,7 @@ tasks:
     - |
       PATH="{{.BUILD_ROOT}}/bin:{{.GO_BUILD_ROOT}}/bin:$PATH"
       make {{.CLI_ARGS}}
+    - tar -czvf e2e-artifacts.tar.gz tests/_e2e_artifacts
```

Comment:
this will only work if the `e2e-tests` target was invoked. For any other target it will cause a failure. An alternative would be to put it in the `vtools/taskfiles/ci.yml` file